### PR TITLE
add another database

### PR DESCRIPTION
**Pull request by itkg-rboessel**

### CODE REVIEW

File: en/developer_guide/other_database_type.rst [hunk 1]
```diff
@@ -0,0 +1,44 @@
+Using other database type
+=========================
+
+Context
+-------
+
+Open orchestra use originally a MongoDb database. If you want to use another type of database like SQL,
```
Comment (ngilain) [inline en/developer_guide/other_database_type.rst]: Open Orchestra use by default a MongoDb database.

File: en/developer_guide/other_database_type.rst [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+--------------------------------------
+
+By default, open orchestra use Doctrine ODM to manage object. Developers can define another document manager implementing
+``Doctrine\Common\Persistence\ObjectManager`` with a change in configuration
+
+
+.. code-block:: yaml
+
+  open_orchestra_base_bundle:
+      object_manager: path/to/objectManager
```
Comment (ngilain) [inline en/developer_guide/other_database_type.rst]: Path is more for a file, I think you need the namespace here, isn't it ?

File: en/developer_guide/other_database_type.rst [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+
+Override mongo documents and repositories
+-----------------------------------------
+
+Some bundles have dependencies to mongo. To replace them, their are several solution depending of the type of dependencies.
```
Comment (ngilain) [inline en/developer_guide/other_database_type.rst]: There is several solutions

File: en/developer_guide/other_database_type.rst [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+All the bundle need to be replace
```
Comment (ngilain) [inline en/developer_guide/other_database_type.rst]: Bundles needing to be replaced

File: en/developer_guide/other_database_type.rst [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The ``open-orchestra-model-bundle`` has mongo dependencies. It need to be replace by a new bundle using
```
Comment (ngilain) [inline en/developer_guide/other_database_type.rst]: to be replaced

File: en/developer_guide/other_database_type.rst [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+``open-orchestra-model-interface``.
+
+``open-orchestra-base-api-bundle`` need an api model bundle to work. In ODM version,
+``open-orchestra-base-api-mongo-model-bundle`` work. To use another database, you need to create a model to
+ your base type and add it to the composer.json
+
+A model bundle need to be replace
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+``open-orchestra-workflow-function-bundle`` and ``open-orchestra-media-bundle`` are call with a specific model
```
Comment (ngilain) [inline en/developer_guide/other_database_type.rst]: are called

File: en/developer_guide/other_database_type.rst [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The ``open-orchestra-model-bundle`` has mongo dependencies. It need to be replace by a new bundle using
+``open-orchestra-model-interface``.
+
+``open-orchestra-base-api-bundle`` need an api model bundle to work. In ODM version,
+``open-orchestra-base-api-mongo-model-bundle`` work. To use another database, you need to create a model to
+ your base type and add it to the composer.json
+
+A model bundle need to be replace
```
Comment (ngilain) [inline en/developer_guide/other_database_type.rst]: What do you mean here ? I don't understand the title

**Comment by itkg-rboessel** [comment]
@itkg-ngilain @itkg-alavieille

File: en/developer_guide/other_database_type.rst [hunk 8]
```diff
@@ -0,0 +1,50 @@
...
+Context
+-------
+
+Open orchestra use by default a MongoDb database. If you want to use another type of database like SQL,
+you need to do some work for now.
+
+Change object manager in configuration
+--------------------------------------
+
+By default, open orchestra use Doctrine ODM to manage object. Developers can define another document manager implementing
```
Comment (ngilain) [inline en/developer_guide/other_database_type.rst]: to manage objects

File: en/developer_guide/other_database_type.rst [hunk 9]
```diff
@@ -0,0 +1,50 @@
...
+
+.. code-block:: yaml
+
+  open_orchestra_base_bundle:
+      object_manager: namespace/of/objectManager
+
+Override mongo documents and repositories
+-----------------------------------------
+
+Some bundles have dependencies to mongo. To replace them, their is several solution depending of the type of dependencies.
```
Comment (ngilain) [inline en/developer_guide/other_database_type.rst]: there is several solutions

File: en/developer_guide/other_database_type.rst [hunk 10]
```diff
@@ -0,0 +1,50 @@
...
+
+All the bundle need to be replace
```
Comment (ngilain) [inline en/developer_guide/other_database_type.rst]: Not sure of what you want to tell. Maybe something simplier if it's the meaning: "Some packages need to be completly replaced"

File: en/developer_guide/other_database_type.rst [hunk 11]
```diff
@@ -0,0 +1,50 @@
...
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The ``open-orchestra-model-bundle`` and ``open-orchestra-base-api-mongo-model-bundle`` have mongo
+dependencies and need to be replace by your bundle.
```
Comment (ngilain) [inline en/developer_guide/other_database_type.rst]: to be replaced by your packages

File: en/developer_guide/other_database_type.rst [hunk 12]
```diff
@@ -0,0 +1,50 @@
...
+
+A repository in the package need to be replace
```
Comment (ngilain) [inline en/developer_guide/other_database_type.rst]: Some bundles need to be replaced

File: en/developer_guide/other_database_type.rst [hunk 13]
```diff
@@ -0,0 +1,50 @@
...
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Some bundles are call with a specific model bundle dependant to mongo in their dependencies. To use
```
Comment (ngilain) [inline en/developer_guide/other_database_type.rst]: are called

File: en/developer_guide/other_database_type.rst [hunk 14]
```diff
@@ -0,0 +1,50 @@
...
+another database, these bundles must not be activate in the appKernel and replace by yours.
```
Comment (ngilain) [inline en/developer_guide/other_database_type.rst]: activated [...] replaced

**Comment by itkg-rboessel** [comment]
@itkg-ngilain @itkg-alavieille

File: en/developer_guide/other_database_type.rst [hunk 15]
```diff
@@ -0,0 +1,50 @@
+Using other database type
+=========================
+
+Context
+-------
+
+Open orchestra use by default a MongoDb database. If you want to use another type of database like SQL,
```
Comment (ngilain) [inline en/developer_guide/other_database_type.rst]: uses

File: en/developer_guide/other_database_type.rst [hunk 16]
```diff
@@ -0,0 +1,50 @@
...
+you need to do some work for now.
+
+Change object manager in configuration
+--------------------------------------
+
+By default, open orchestra use Doctrine ODM to manage objects. Developers can define another document manager implementing
```
Comment (ngilain) [inline en/developer_guide/other_database_type.rst]: uses

File: en/developer_guide/other_database_type.rst [hunk 17]
```diff
@@ -0,0 +1,50 @@
...
+
+.. code-block:: yaml
+
+  open_orchestra_base_bundle:
+      object_manager: namespace/of/objectManager
+
+Override mongo documents and repositories
+-----------------------------------------
+
+Some bundles have dependencies to mongo. To replace them, their is several solutions depending of the type of dependencies.
```
Comment (ngilain) [inline en/developer_guide/other_database_type.rst]: there is

File: en/developer_guide/other_database_type.rst [hunk 18]
```diff
@@ -0,0 +1,50 @@
...
+
+Some packages need to be completely replaced
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The ``open-orchestra-model-bundle`` and ``open-orchestra-base-api-mongo-model-bundle`` have mongo
+dependencies and need to be replaced by your packages.
+
+Some bundles in the package need to be replace
```
Comment (ngilain) [inline en/developer_guide/other_database_type.rst]: replaced

**Comment by itkg-rboessel** [comment]
@itkg-alavieille @itkg-ngilain

File: en/developer_guide/other_database_type.rst [hunk 19]
```diff
@@ -0,0 +1,50 @@
...
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The ``open-orchestra-model-bundle`` and ``open-orchestra-base-api-mongo-model-bundle`` have mongo
+dependencies and need to be replaced by your packages.
+
+Some bundles in the package need to be replaced
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Some bundles are called with a specific model bundle dependant to mongo in their dependencies. To use
+another database, these bundles must not be activated in the ``appKernel`` and replaced by yours.
```
Comment (ngilain) [inline en/developer_guide/other_database_type.rst]: and must be replaced

**Comment by ngilain** [comment]
@itkg-alavieille , @itkg-nanne : doc updated

**Comment by veolys-nanne** [comment]
:+1:

File: en/developer_guide/other_database_type.rst [hunk 20]
```diff
@@ -0,0 +1,54 @@
...
+-------
+
+Open orchestra uses by default a MongoDB database. If you want to use another type of database like
+SQL, you will need to tune some settings and to replace some files.
+
+Configuration: object manager
+-----------------------------
+
+By default, Open Orchestra uses Doctrine ODM to map database documents to objects. Developers can
+define another document manager, the only condition is to implement
```
Comment (alavieille) [inline en/developer_guide/other_database_type.rst]: must not can
This sentence is weird : 
Developers must define another document manager which implement Doctrine\Common\Persistence\ObjectManager like EntityManager for instance in configuration

File: en/developer_guide/other_database_type.rst [hunk 21]
```diff
@@ -0,0 +1,54 @@
...
+Override mongo documents and repositories
+-----------------------------------------
+
+Some bundles have dependencies to mongoDB. The way to replace them depends on the type of
+dependencies.
+
+Complete replacement of the package
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The ``open-orchestra-model-bundle`` and ``open-orchestra-base-api-mongo-model-bundle`` have mongo
```
Comment (alavieille) [inline en/developer_guide/other_database_type.rst]: what do you think of : 
The open-orchestra-model-bundle and open-orchestra-base-api-mongo-model-bundle are an implementation of entities used in Open Orchestra for MongoDB and need to be replaced by your implementation for another database

**Comment by ngilain** [comment]
@itkg-alavieille : update done

**Comment by alavieille** [comment]
:+1:

**Pull request merged**